### PR TITLE
Extension analytics

### PR DESCRIPTION
**Pull request by CubeOfCheese**
Adds google analytics to the popup

### CODE REVIEW

**Comment by CubeOfCheese** [comment]
I don't know for sure that I did this right, so please give it a good review Linda

**Comment by CubeOfCheese** [comment]
closes #131

File: Extension/popup.js [hunk 1]
```diff
@@ -216,3 +216,14 @@ function somethingWrong() {
         }
     });
 }
+
+// GOOGLE ANALYTICS
```
Comment (lindaicing) [inline Extension/popup.js:220]: Should this be at the top of the page? Website analytics are usually placed at the top

Comment (CubeOfCheese) [reply Extension/popup.js:220]: I don't think it makes a difference. I would prefer it at the bottom because the top of the file is more accessible to us

Comment (CubeOfCheese) [reply Extension/popup.js:220]: I've seen it at the top when you link it in html pages, but this is javascript. Is it at the top of javascript files usually too?

Comment (lindaicing) [reply Extension/popup.js:220]: Sounds good!

**Review by lindaicing — APPROVED** [review]

**Pull request merged**